subiquity/common/types.py

```diff
@@ -340,6 +341,14 @@ def is_core_boot(self) -> bool:
                         GuidedCapability.CORE_BOOT_PREFER_ENCRYPTED,
                         GuidedCapability.CORE_BOOT_PREFER_UNENCRYPTED]
 
+    def supports_manual_customization(self) -> bool:
+        # After posting this capability to guided_POST, is it possible
+        # for the user to customize the layout further?
```

Comment:
I believe so but I bet there are strange corner case bugs

Reply:
Um this comment is an explanation, not a question :-)